### ISSUE TITLE
Async component will be loaded at the first time

### ISSUE THREAD

**Issue by SidKwok** [theKashey/react-imported-component#15]
App.jsx
```jsx
import React, { Component } from 'react';
import Imported from 'react-imported-component'
import './App.css';

const AsyncComponent = Imported(() => import('./AsyncComponent'))

class App extends Component {
  state = {
    show: false
  }
  handleClick() {
    this.setState({show: !this.state.show});
  }
  render() {
    return (
      <div className="App">
        <button onClick={() => this.handleClick()}>switch</button>
        {
          this.state.show && <AsyncComponent />
        }
      </div>
    );
  }
}

export default App;

```

AsyncComponent.jsx
```jsx
import React from 'react'

export default () => (
    <div>async</div>
);

```

output:
![f914131c815f2d7339d3a01b8a2f4293](https://user-images.githubusercontent.com/13268747/39915021-56cd7926-5539-11e8-99e0-f886e26f7da8.png)

env:

node: 8.10.0
react: 16.3.2
react-imported-component: 4.3.2

It works with 1.0.4

**Comment by theKashey** [comment]
Hm, I'll check.

**Comment by theKashey** [comment]
Fixed in v4.3.3
